pkg/annotations/annotations.go

```diff
@@ -337,6 +337,13 @@ const (
 
 	// VSMBNoDirectMap specifies that no direct mapping should be used for any VSMBs added to the UVM.
 	VSMBNoDirectMap = "io.microsoft.virtualmachine.wcow.virtualSMB.nodirectmap"
+
+	// WCOWIsolationType specifies the isolation type for the WCOW UVM.
+	// Supported values are: "SNP", "VBS", and "GuestStateOnly".
+	WCOWIsolationType = "io.microsoft.virtualmachine.wcow.isolationtype"
+
```

Comment:
My PR that merged recently added these annotations, so we don't need these anymore in here.